feature-libs/quote/components/actions/by-role/quote-actions-by-role.module.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { QuoteCoreConfig } from '@spartacus/quote/core';
+import { QuoteActionType } from '@spartacus/quote/root';
+import {
+  AuthGuard,
+  CmsConfig,
+  I18nModule,
+  provideDefaultConfig,
+} from '@spartacus/core';
+import { QuoteActionsByRoleComponent } from './quote-actions-by-role.component';
+
+@NgModule({
+  imports: [CommonModule, I18nModule],
+  providers: [
+    provideDefaultConfig(<QuoteCoreConfig>{
```

Comment:
@Uli-Tiger I think we should put that into the core entry point as the normalizers access it. A bit weird that a component defines this configuration. Do you agree? That would be in core/config then

Reply:
agree